Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -200,20 +213,23 @@ const CreateNewMaintenanceWindow = () => {
       component: (
         <Stack
           className="add-monitors-fields"
-          sx={{ width: "60%", maxWidth: "380px" }}
+          sx={{ width: "60%", maxWidth: 380 }}
           gap={theme.spacing(5)}
         >
-          <Field
-            id="add-monitors"
-            placeholder="Start typing to search for current monitors"
-            value={values.AddMonitors}
-            onChange={(e) => handleChange(e, "AddMonitors")}
+          <AutoCompleteField
+            sx={{ marginLeft: "auto" }}
+            options={monitorOptions}
+            width={380}
+            autoCompleteValue={autoCompleteValue}
+            setAutoCompleteValue={setAutoCompleteValue}
+            autoCompleteInputValue={autoCompleteInputValue}
+            setAutoCompleteInputValue={setAutoCompleteInputValue}
             error={errors.addMonitors}
```

Comment:
Is it possible to restrict the input to options that exist?  Right now I can enter whatever I want as long as it begins with an option in the list:

![image](https://github.com/user-attachments/assets/15e2b901-3a2d-43cf-9840-80fd165d4c2f)

And this will be accepted as acceptable input.  It would be best if this could be restricted.

Also if I enter something that is not acceptable it is not rejected, an error is thrown and is unhandled:

![image](https://github.com/user-attachments/assets/58878527-8dda-4b0d-9f79-77a28ae1431a)

This needs to be addressed as well and will probably be resolved by fixing the above issue

Reply:
I somehow fixed it.
The text field is disabled now!
the user might be able to type, but only an option from the list is selected.
This is what could be done by attributes that was provided with components.